src/structures/BaseGuildEmoji.js

```diff
@@ -32,21 +32,21 @@ class BaseGuildEmoji extends Emoji {
 
     /**
      * Whether or not this emoji requires colons surrounding it
-     * @type {boolean}
+     * @type {?boolean}
```

Comment:
`?boolean` [indicates nullable](https://jsdoc.app/tags-type.html), the property may actually be undefined

Reply:
Huh, weird, looks like other properties such as [Guild#approximateMemberCount](https://github.com/discordjs/discord.js/blob/master/src/structures/Guild.js#L323) can be undefined yet the type is still `{?number}`, any reason why that would be?
I could just be majorly overthinking this but what would be the correct alternative to `boolean` to show undefined in this scenario?

Reply:
well, according to the jsdoc docs that is also typed incorrectly it would seem? the correct type should be `boolean|undefined`i believe

Reply:
Huh, that's what I thought too but I don't see djs use that consistently in their typings, they usually just use `?type` for undefined or null - I could definitely change it but I would rather be consistent with the library itself even if, according to JSDoc, it is wrong.